Sources/ApolloCodegenLib/TemplateString+AvailabilityDeprecated.swift

```diff
@@ -6,25 +6,44 @@ extension TemplateString.StringInterpolation {
   ) {
     guard
       config.options.warningsOnDeprecatedUsage == .include,
-      let deprecationReason = deprecationReason
+      let escapedDeprecationReason = deprecationReason?.escapedDoubleQuotes()
     else {
       removeLineIfEmpty()
       return
     }
 
-    let deprecationReasonLines = deprecationReason
-      .split(separator: "\n", omittingEmptySubsequences: false)
+    if escapedDeprecationReason.firstIndex(of: "\n") != nil {
```

Comment:
I think we can actually get rid of this multi-line processing and skip replacing the `\n` characters. I don't see any benefit to having multi-lines in the `@avialable` annotation and Xcode correctly parses the `\n` into new lines when showing it to the user anyways.

<img width="856" alt="Screenshot 2023-04-11 at 11 18 18 AM" src="https://user-images.githubusercontent.com/146856/231254147-9685f468-4458-4744-b7aa-e2f8e385a351.png">